src/domain/collaboration/templates/CollaborationTemplatesLibrary/CollaborationTemplatesLibrary.tsx

```diff
@@ -16,7 +16,7 @@ import { Identifiable, Identifiables } from '../../../../core/utils/Identifiable
 import DialogWithGrid from '../../../../core/ui/dialog/DialogWithGrid';
 import { identity } from 'lodash';
 import Spacer from '../../../../core/ui/content/Spacer';
-import ConfirmationDialog from '../../../../core/ui/dialogs/ConfirmationDialog';
+import CommunityGuidelinesConfirmDialog from '../../communityGuidelines/CommunityGuidelinesTemplateLibrary/CommunityGuidelinesConfirmDialog';
```

Comment:
I don't fully like this, you have made `CollaborationTemplatesLibrary` which was a generic thing for any type of template, dependent on a CommunityGuidelines component. 
Would it be easy to pass a `confirmationDialog` ComponentType that you can pass from outside and gets open or closed from here?